code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -131,6 +131,10 @@
 			continue
 		if(possible_target.current.stat == DEAD)
 			continue
+		// BUBBER EDIT ADDITION - Round Removal OPT-IN (Only security and command can be targeted)
+		if (!possible_target.assigned_role?.heretic_sac_target)
```

Comment:
This means heretics will never target non-sec/command, please describe all your changes in the PR

Reply:
I decided since heretics are already hugboxxed I'll step back on this for now, eventually we can un-hugbox heretic sacs and make it target anyone opted in